css/portal-dashboard/num-students.less

```diff
@@ -0,0 +1,23 @@
+@import "./variables";
+
+.numStudentsContainer {
+    width: 168px;
+    height: 30px;
+    border-radius: 15px 0px 0px 15px;
+    background-color: @cc-teal-light7;
+    position: absolute;
+    top: 210px;
+    left: 72px;
+    font-size: 16px;
+    font-family: lato, arial, helvetica, sans-serif;
+    letter-spacing: 0;
+    color: @cc-charcoal;
+}
+.containerLabel {
+    font-weight: bold;
+    padding-left:20px;
+}
+.numStudentsContainer span {
+    position: relative;
+    top: 6px;
```

Comment:
I like to use flex boxes to center thing vertically and horizontally.  Here you might try just setting `line-height: 30px;` instead of `top: 6px` and see if it looks like the text is in the true vertical center.  On my system/screen that looks a pixel or so better than setting the top position.  Not sure if you see the same improvement.